src/main/resources/org/broadinstitute/hellbender/utils/plotter/CNV_plotting_library.R

```diff
@@ -47,8 +47,36 @@ SetUpPlot = function(y.lab, y.min, y.max, x.lab, label_chromosomes, num_chromoso
   }
 }
 
+## Chromosome Background for data
+SetUpPlotPerSeg = function(y.lab, y.min, y.max, x.lab, label.x, x.axt, segs){
+  #plot in megabases
+  scaling = 1000000
+  start = segs[1, "Start"]
+  end = segs[nrow(segs), "End"]
+  chr = segs[1,"Chromosome"]
+  suppressWarnings( par( mar=c(3.1,3.6,0.1,0),mgp=c(2,-0.2,-1.1) ) )
+  options(scipen=3)
+  plot(1, type="n", xlab="", ylab="", xlim=c(start, end), ylim=c(y.min, y.max), bty='n', xaxt='n')
+  scaled_start = start/scaling
+  scaled_end = end/scaling
+  labels=signif(seq(scaled_start, scaled_end, by=((scaled_end-scaled_start)/10)), 2)
+  sigs = 0
```

Comment:
It's a surprising amount of code. Requires stuff like
``` floor(abs(increment)/(10^floor(log10(abs(increment))))) ```
and requires considering a number of cases (increment more than zero or less than zero, start less than or more than zero, etc.)

The loop is instant to run and guaranteed to work, so unless there's an easier way of figuring out what digits will be necessary that I'm not thinking of, I think the loop should be acceptable.